src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    public static final int MIN_LENGTH = 1;
+    public static final int MAX_LENGTH = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        return switch (figureNumber) {
+            case 0 -> circle();
+            case 1 -> isoscelesTrapezoid();
+            case 2 -> rectangle();
+            case 3 -> rightTriangle();
+            default -> square();
+        };
+    }
+
+    public Figure circle() {
```

Comment:
```suggestion
    private Figure circle() {
```
all methods that creating figures should be private, except default type